addons/web/static/src/views/view_dialogs/select_create_dialog.js

```diff
@@ -51,6 +51,17 @@ export class SelectCreateDialog extends Component {
         }
     }
 
+    async unselect() {
+        if (this.props.onUnselect) {
```

Comment:
I expected to see this in a props definition somewhere

Reply:
Which props definition ? :innocent:

Reply:
Will be added in forward port as not present in 16.0 and so avoid error like :

> Invalid props for component 'SelectCreateDialog': unknown key 'title', unknown key 'noCreate', unknown key 'multiSelect', unknown key 'resModel', unknown key 'context', unknown key 'domain', unknown key 'onSelected', unknown key 'onCreateEdit', unknown key 'dynamicFilters', unknown key 'close'